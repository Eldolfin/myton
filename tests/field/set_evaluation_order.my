undefined1.bar = undefined2 # expect error: undefined1 is not defined
