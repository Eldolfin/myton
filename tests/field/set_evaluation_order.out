[31merror[error]: Undefined variable 'undefined1'
----- Traceback -----
  â”Œâ”€ <unknown>:1:10
  | 
1 | undefined1.bar = undefined2 # expect error: undefined1 is not defined
  |           ^
[0m